sgeop/geometry.py

```diff
@@ -59,6 +61,10 @@ def angle_between_two_lines(line1, line2):
     # (lines touch in this point)
     points = collections.Counter([a, b, c, d])
 
+    lines_share_vertex = max(points.values()) > 1
+    if not lines_share_vertex:
+        raise ValueError("Input lines do not share a vertex.")
```

Comment:
As above.

Reply:
As above.

Reply:
Here the returned value is trickier. But let's do 0 for simplicity as well?

Reply:
> Here the returned value is trickier.

Yeah... 

> But let's do 0 for simplicity as well?

Sounds like a plan. At least with a healthy warning we can retain our sanity while debugging later if it becomes a problem.